lib/adapters/stdio/read.js

```diff
@@ -8,7 +8,7 @@ var Read = Juttle.proc.source.extend({
     procName: 'read stdio',
 
     initialize: function(options, params, location, program) {
-        var allowed_options = ['timeField', 'format', 'formatOptions'];
+        var allowed_options = ['timeField', 'format', 'pattern'];
```

Comment:
We should probably throw an error if the user specifies a `-pattern` option when the format isn't `grok`.